<?xml version="1.0"?>

<launch>

  <node name="salih_marangoz_thesis_node" pkg="salih_marangoz_thesis" type="ceres_ik_node">

  </node>


</launch>
